wms_connector/models/stock_warehouse.py

```diff
@@ -0,0 +1,210 @@
+# Copyright 2023 Akretion
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from odoo import fields, models
+
+FILTER_FIELDNAMES = [
+    "wms_export_product_filter_id",
+    "wms_product_sync_filter_id",
+    "wms_export_picking_in_filter_id",
+    "wms_export_picking_out_filter_id",
+]
+MAPPINGS = {
+    "export": {
+        "fieldname_task": "wms_export_task_id",
+        "fieldname_cron": "wms_export_cron_id",
+        "filetype": "export",
+        "name_fragment": "exports (products, awaiting receptions, preparation orders",
+        "code": "wh = env['stock.warehouse'].browse({0})\n"
+        "wh.{1}.scheduler_export("
+        '"wms.product.sync", wh._wms_domain_for("wms_product_sync")'
+        ")\n"
+        "wh.{1}.scheduler_export("
+        '"stock.picking", wh._wms_domain_for("pickings_in")'
+        ")\n"
+        "wh.{1}.scheduler_export("
+        '"stock.picking", wh._wms_domain_for("pickings_out")'
+        ")",
```

Comment:
env["wms.product.sync"]._schedule_export(warehouse)

on the wms.product.sync

def _schedule_export(self, warehouse):
     warehouse._sync_wms_product()
     return super()._schedule_export(warehouse)





env["stock.picking"]._schedule_export(warehouse, domain=wh._wms_domain_for("pickings_in") )

env["stock.picking"]._schedule_export(warehouse, domain=wh._wms_domain_for("pickings_out"))